src/SamplesApp/SamplesApp.UITests/SamplesApp.UITests.csproj

```diff
@@ -5,11 +5,10 @@
 	</PropertyGroup>
 
 	<ItemGroup> 
-		<PackageReference Include="FluentAssertions" Version="5.10.3" />
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+		<PackageReference Include="FluentAssertions" />
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
 		<PackageReference Include="NUnit" />
 		<PackageReference Include="NUnit3TestAdapter" /> 
-		<PackageReference Include="Xamarin.UITest" Version="3.0.12" />
 		<PackageReference Include="Uno.UITest" />
```

Comment:
Are these changes intentional?

Reply:
It is.

Reply:
@davidjohnoliver it was an update when it was done ;-)